lib/model.yaml

```
@@ -4,9 +4,8 @@ Tab:
   charts: "array Chart"
 
 Chart:
-  data_path: DataPath
+  data_collection: "string"
```

I know it's a bit tedious, but the Dart recommended style is to use camel case capitalisation. I think it's preferable for the config to be in underscore though, so let's rename this to camelcase for the generated dart model:
```suggestion
  data_collection: "string dataCollection"
```

---

Ahh this is cool, didnt know this was an option. Fixed